### ISSUE TITLE
Curl->download() - supplied argument is not a valid File-Handle resource

### ISSUE THREAD

**Issue by antons-** [php-curl-class/php-curl-class#159]
When I attempt to use this function I receive the following warning. The file still downloads successfully though:
```
[Warning] curl_setopt(): supplied argument is not a valid File-Handle resource
* curl_setopt(Resource id #49,10001,)
  Curl.php:430
* Curl\Curl->setOpt(10001,)
  Curl.php:195
```

This is the code I'm using:
```
$curl = new Curl();
$curl->setOpt(CURLOPT_ENCODING, 'gzip');
$curl->setOpt(CURLOPT_SSL_VERIFYPEER, 0);
$curl->setOpt(CURLOPT_SSL_VERIFYHOST, 0);
$curl->download('https://domain.com/controller/action', '/var/www/project/assets/file.tar.gz');

if ($curl->error) {
    usererror('Error: ' . $curl->error_code . ': ' . $curl->error_message);
}
$curl->close();
```

If I delete line 195 in ```Curl.php```, I no longer see the warning.
```
$this->setOpt(CURLOPT_FILE, STDOUT);
```

I'm not sure if this is a bug or I'm doing something wrong? If it helps I'm running PHP 5.5.9 and using php-curl-class 3.4.3.

**Comment by zachborboa** [comment]
Hi, thank you for the report. I'm able to reproduce on 5.5.9. I will investigate.

**Comment by zachborboa** [comment]
Fixed in 343252cfdfd6111d2d9f367e81dbe1c52b5c6085. Can you try again with the latest version?

**Comment by antons-** [comment]
Yes, it's working as expected now (no warning), thanks for the quick fix :)
